hassio/src/system/hassio-supervisor-log.ts

```diff
@@ -0,0 +1,77 @@
+import "@material/mwc-button";
+import "@polymer/paper-card/paper-card";
+import {
+  css,
+  CSSResult,
+  customElement,
+  html,
+  LitElement,
+  property,
+  TemplateResult,
+  query,
+} from "lit-element";
+
+import { ANSI_HTML_STYLE, parseTextToColoredPre } from "../ansi-to-html";
+import { hassioStyle } from "../resources/hassio-style";
+import { haStyle } from "../../../src/resources/styles";
+import { HomeAssistant } from "../../../src/types";
+
+@customElement("hassio-supervisor-log")
+class HassioSupervisorLog extends LitElement {
+  @property() public hass!: HomeAssistant;
+  @query("#content") private _logContet!: any;
+
+  public connectedCallback(): void {
+    super.connectedCallback();
+    this._loadData();
+  }
+
+  public render(): TemplateResult | void {
+    return html`
+      <paper-card>
+        <div class="card-content" id="content"></div>
+        <div class="card-actions">
+          <mwc-button @click=${this._refresh}>Refresh</mwc-button>
+        </div>
+      </paper-card>
+    `;
+  }
+
+  static get styles(): CSSResult[] {
+    return [
+      haStyle,
+      hassioStyle,
+      ANSI_HTML_STYLE,
+      css`
+        pre {
+          white-space: pre-wrap;
+        }
+      `,
+    ];
+  }
+
+  private _loadData(): void {
+    this.hass.callApi("GET", "hassio/supervisor/logs").then(
```

Comment:
Move to `data/hassio.ts`.

We need to break up the `data/hassio.ts` file into `data/hassio/supervisor.ts`, `data/hassio/addon.ts` etc.